core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -101,6 +111,28 @@ oppia.directive('tutorCard', [
             }
           };
 
+          $scope.consumeSolution = function() {
+            playerTranscriptService.addNewInput(
+              'Please show me the answer.', true);
+            var solution = SolutionManagerService.consumeSolution();
+            var interactionId = stateInteractionIdService.savedMemento;
+            var answer = '';
+            if (interactionId === 'MultipleChoiceInput') {
+              answer = (
+                responsesService.getAnswerChoices()[solution.correctAnswer]
+                .label);
+            } else if (interactionId === 'MathExpressionInput') {
+              answer = solution.correctAnswer.latex;
+            } else {
+              answer = solution.correctAnswer;
+            }
+            var answerIsExclusive = solution.answerIsExclusive ? 'Only' : 'One';
+            var response = (
+            answerIsExclusive + ' answer is:<br>' + answer +
+            '<br><br>Explanation:<br>' + solution.explanation);
```

Comment:
I think you can get rid of "Explanation:". I think it seems clear enough that it's an explanation with the line break separating it from the solution. It'll also help it feel a tad more conversational.